### PR TITLE
Prevent invalid element name

### PR DESCRIPTION
**Pull request by emiliodeg**
Prevent create form element with empty name value like ''

Hello!

* Type: bug fix
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change:

Thanks

### CODE REVIEW

File: phalcon/forms/element.zep [hunk 1]
```diff
@@ -60,6 +60,12 @@ abstract class Element implements ElementInterface
 	 */
 	public function __construct(string name, var attributes = null)
 	{
+		let name = trim(name);
+		
+		if empty name {
+			throw new \InvalidArgumentException('Form element name is required');
```
Comment (sergeyklay) [inline phalcon/forms/element.zep]: `throw new \InvalidArgumentException('...');` **=>**`throw new \InvalidArgumentException("...");`

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@emiliodeg What do you want to fix? Could you please describe a bit more the issue? Also can you  create a small test which will cover code you have changed? Thank you for contributing.

**Comment by emiliodeg** [comment]
Hi @sergeyklay I trying to prevent this posible error

```php 
new \Phalcon\Form\Element\Text(''); // empty name
new \Phalcon\Form\Element\TextArea('      '); // empty name
```

**Comment by sergeyklay** [comment]
@emiliodeg Anyway you have to use `" "` instead of `' '`.

**Review by sergeyklay — APPROVED** [review]
@emiliodeg Looks good to me. Could you please update the CHANGELOG.md

**Pull request merged**